CONTRIBUTING.md

```diff
@@ -330,54 +368,55 @@ node seed
 npm run develop
 ```
 
-### Setup freeCodeCamp using Docker
-#### Isolated
+## Setup using Docker (Alternative Method)
+
+### Isolated
+
 Use this if you just want to work on freeCodeCamp.
 
 You will need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing the commands below.
 
 Setup:
-```bash
+
+```shell
 docker-compose run --rm freecodecamp npm install
 docker-compose run --rm freecodecamp npm run only-once
 ```
 
 Run:
-```bash
+
+```shell
 docker-compose up
 ```
 
-#### Shared
+### Shared
+
 Use this if you want to work on other services that will run alongside
 of freeCodeCamp, using the  database directly. An example is the [open-api](https://github.com/freeCodeCamp/open-api) project.
 
-```bash
+```shell
 docker-compose -f docker-compose.yml -f docker-compose-shared.yml up
 ```
 
-
 ### Creating a New User
+
 To create a new user, you will need to perform the following steps:
-* run MailHog if you haven't set up yet check [set-up-mailhog](#set-up-mailhog)
-* run `npm run develop` and navigated to [http://localhost:3000](http://localhost:3000/)
-* Click over the Sign-Up link situated on the right corner of the navigation bar.
-* Write your email over the input and press the button "get a sign in link" this will send an email with the login URL into MailHog.
-* The last step is to check your inbox in MailHog for a new email from "team@freecodecamp.org" click over the URL inside of it you will be redirected and logged in into the app.
 
-### Make Changes
+- run MailHog if you haven't set up yet check [set-up-mailhog](#set-up-mailhog)
+- run `npm run develop` and navigated to [http://localhost:3000](http://localhost:3000/)
```

Comment:
```diff
- ...and navigated to...
+ ...and navigate to...
```

And put local host in back ticks to be consistent with other mentions of local host.